src/components/footerIcon/FooterIcon.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import styles from './footerIcon.module.css';
+
+interface IconLinkProps {
+    media: Media[];
+    className?: string;
+}
+
+interface Media {
+    icon: string;
+    url: string;
+}
+
+const FooterIcon = ({ media, className }: IconLinkProps) => (
```

Comment:
it seems like more `FooterIcons` component. You can create another one for this code and name like `FooterIcon` or `SocialIcon` and give props into component

` <a
                key={index}
                href={item.url}
                target="_blank"
                rel="noopener noreferrer"
                className={index === media.length - 1 ? '' : styles.mr_30}
            >
                <img src={item.icon} alt="Icon" />
            </a>`

Reply:
added new component.